site/components/pages/errors/Custom404.tsx

```diff
@@ -13,25 +13,24 @@ export const Custom404 = () => {
     <div className={styles.errorPageWrapper}>
       <PageHead
         title={t({
-          id: "error.page.404.title",
+          id: "shared.404",
           message: "404 - Page Not Found",
         })}
         mediaTitle={t({
-          id: "error.page.404.head.title",
+          id: "shared.404",
```

Comment:
What are the rules for using "shared"?
This page is used only once, that is why I am wondering 🤔